index.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta content="width=device-width, initial-scale=1.0" name="viewport" />
+    <title>Resume</title>
+    <link href="/assets/css/main.css" rel="stylesheet" type="text/css" />
+    <link
+      href="https://fonts.googleapis.com/css?family=Merriweather|Oswald&display=swap"
+      rel="stylesheet"
+    />
+  </head>
+  <body>
+    <div class="flex-container-column" id="container">
+      <header id="header">
+        <div class="flex-container-row">
+          <div id="image"></div>
+          <div>
+            <h2 id="name"></h2>
+            <h3 id="label"></h3>
+            <p id="contact">
+              <span id="email"></span>
+              <span id="phone"></span>
+              <span id="website"></span>
+              <span id="address"></span>
+            </p>
+            <p id="profiles"></p>
+          </div>
+        </div>
+      </header>
+      <div id="summary"></div>
+      <div id="skills">
+        <h3>Skills</h3>
```

Comment:
just a stylistic thing. I thought general practice nowadays is to avoid using ID's unless it's necessary. Personally I try to use unique class names for naming. It helps with finding patterns in your styling if you have different parts that have the same information.